homeassistant/components/plex/config_flow.py

```diff
@@ -263,3 +312,25 @@ def __init__(self, config_entry):
                 }
             ),
         )
+
+
+class PlexAuthorizationCallbackView(HomeAssistantView):
+    """Handle callback from external auth."""
+
+    url = AUTH_CALLBACK_PATH
+    name = AUTH_CALLBACK_NAME
+    requires_auth = False
+
+    async def get(self, request):
+        """Receive authorization confirmation."""
+        from aiohttp import web_response
+
+        hass = request.app["hass"]
+        await hass.config_entries.flow.async_configure(
+            flow_id=request.query["flow_id"], user_input=None
```

Comment:
What string is used to validate the authentication? There's no code?

Reply:
Nope. Callback is just a notification that it's ready to query the endpoint that actually provides the token.

Reply:
Token is grabbed in the following step, `async_step_obtain_token()`.

Reply:
I see that the code is already retrieved by the library in the first step.